app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,37 @@
+<nav class="navbar navbar-expand-lg bg-body-secondary rounded mb-3 ">
+  <div class="container-fluid">
+    <%= link_to 'Home', recipes_path, class: 'navbar-brand' %>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <%= link_to 'Public Recipes', public_recipes_path, class: 'nav-link' %>
+        </li>
+        <li class="nav-item">
+          <%= link_to 'All invetories', inventories_path, class: 'nav-link' %>
+        </li>
+        <li class="nav-item">
+          <%= link_to 'Foods', foods_path, class: 'nav-link' %>
+        </li>
+        <li class="nav-item">
```

Comment:
Great job my friend this navigation menu looks amazing!
I recommend that you add another button for `all recipes` that redirects you to `recipes#index` as that is one of the wireframes required